### ISSUE TITLE
Page edge follows the finger

### ISSUE THREAD

**Issue by choristery** [oleksandrbalan/pagecurl#26]
The edges of the rolled-up pages do not seem to follow the finger's position on the screen. This gives users a sense of being out of control. When the finger is pressed and begins to move, the raised edge of the page should always follow the finger's position. This would provide a better sense of control.

ex：

[YouTube Video](https://www.youtube.com/shorts/8lnNZBrIl-s?si=sS4xvlPrM-RyvWR2)

**Comment by oleksandrbalan** [comment]
Could I ask you again to check `feature/pointer-behavior` branch and video example in #27 ? 🙏 

This behavior could be set via `pointerBehavior` param:

```kotlin
PageCurl(
    count = pages.size,
    config = rememberPageCurlConfig(
        dragInteraction = PageCurlConfig.GestureDragInteraction(
            pointerBehavior = PageCurlConfig.DragInteraction.PointerBehavior.PageEdge
        )
    )
) { index ->
    HowToPage(index, pages[index])
}
```

**Comment by choristery** [comment]
I tried it and think it's absolutely perfect. 🎉

**Comment by choristery** [comment]
May I ask when this branch will be merged and a new version will be released?

**Comment by oleksandrbalan** [comment]
Sorry for the late reply, I totally forgot to process it further 🤦‍♂️ 

Thank you for your time and testing, and also a reminder, I will try to merge it today at evening and release a new version 🤞

**Comment by oleksandrbalan** [comment]
Released in [v1.5.0](https://github.com/oleksandrbalan/pagecurl/releases/tag/v1.5.0) 🎉
